pom.xml

```diff
@@ -85,12 +85,20 @@
         <joda.version>2.9.6</joda.version>
         <kafka.connect.storage.common.version>11.1.0-SNAPSHOT</kafka.connect.storage.common.version>
         <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
-        <netty.version>4.1.68.Final</netty.version>
+        <netty.version>4.1.71.Final</netty.version>
```

Comment:
@snehashisp I do remember having to upgrade tcnative when I had updated netty as they go hand in hand, could you please check if that also needs an upgrade?

Reply:
The netty library brings in the required version of tcnative. In this case it is not being shadowed by another version being brought by some other dependency. Also, don't see there being a property for tcnative only in the pom so I think we are good here.